addons/mail/static/src/core/common/composer.xml

```diff
@@ -14,7 +14,7 @@
                     'p-3': normal,
                     'o-hasSelfAvatar px-3': !env.inChatWindow and thread,
                 }" t-attf-class="{{ props.className }}">
-            <div class="o-mail-Composer-sidebarMain flex-shrink-0" t-if="!compact and props.sidebar">
+            <div class="o-mail-Composer-sidebarMain flex-shrink-0 pt-1" t-if="!compact and props.sidebar">
```

Comment:
It's becoming misaligned, I don't like the change.

I'd suggest 7px as composer padding top/bottom so that avatar has same height as a single line input.